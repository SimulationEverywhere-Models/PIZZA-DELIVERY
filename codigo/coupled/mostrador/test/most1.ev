00:00:00:010 abrir_m 1
00:00:08:000 atiende_m 1
00:00:13:000 atiende_m 2
00:00:20:000 atiende_m 3
00:00:30:000 recibe_ent_m 2
00:00:48:000 recibe_ent_m 3
00:00:50:000 cerrar_m 1
