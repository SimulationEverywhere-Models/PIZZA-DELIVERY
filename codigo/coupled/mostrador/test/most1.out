00:00:00:010 lista_ll_m 1
00:00:08:000 ocupado_m 1
00:00:10:000 ll_error_m 1
00:00:10:000 libre_m 1
00:00:13:000 ocupado_m 1
00:00:15:000 libre_m 1
00:00:15:030 manda_pedido 2
00:00:20:000 ocupado_m 1
00:00:22:000 libre_m 1
00:00:22:030 manda_pedido 3
00:00:30:000 ocupada_ent_m 1
00:00:30:030 dist_out_m 2
00:00:30:030 libre_m_ent 1
00:00:48:000 ocupada_ent_m 1
00:00:48:030 dist_out_m 3
00:00:48:030 libre_m_ent 1
00:00:50:000 cerrado_m 1
